### DISCUSSION TITLE
2021开源趋势报告;开源软件重塑人工智能的规则；等开源之道每周评论 2021 08 22 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#60] [Announcements]
# 2021开源趋势报告;开源软件重塑人工智能的规则；等开源之道每周评论 2021 08 22 - 「开源之道」

https://opensourceway.community/posts/opensource/daily_reading/2021/%E5%BC%80%E6%BA%90%E4%B9%8B%E9%81%93%E6%AF%8F%E5%91%A8%E8%AF%84%E8%AE%BA-2021-08-22/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

全是断言,
之前建议过, 
所有这种优秀资料的解读/推荐/分析/...

都应该汇集在统一的开源思想关系图谱/谱系/光谱/...中

找个简单的自动图谱生成工具, 每次有这种资料追加进去,
慢慢的, 就形成了专有的 开源宇宙

**Reply by lijiangsheng1** [reply]
大妈说的是，可是我不知道如何做～ 我每天的脑子里都是想着如何学习。
